modules/ROOT/pages/using-the-api-console.adoc

```diff
@@ -0,0 +1,34 @@
+= Using the API Console 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The API Console is an interactive component that shows information about REST APIs and their methods, and enables you to test the methods by sending requests. The API Console is located in the API Details page. Endpoints are displayed in alphabetical order.
```

Comment:
Suggest removing most of this and presenting it as a screenshot with brief callouts, if necessary. Focus on the user tasks instead of the UI.